tools/dependabot/generate-dependabot-pr-changesets.ts

```diff
@@ -36,21 +37,29 @@ function processArgs(): Args {
 	if (args[0] === __filename) {
 		args.shift();
 	}
-	if (args.length !== 3) {
+	if (args.length !== 4) {
 		throw new Error(dedent`
 			Incorrect arguments, please provide three arguments:
 			- PR: The number of the current Dependabot PR
 			- Package: The name of the workers-sdk package whose dependencies are being updated
-			- PackageJSON: The path to the package JSON being updated by Dependabot`);
+			- PackageJSON: The path to the package JSON being updated by Dependabot
+			- Changeset prefix: The prefix to go on the front of the filename of the generated changeset.
+		`);
 	}
 	return {
 		prNumber: args[0],
 		packageName: args[1],
 		packageJSONPath: args[2],
+		changesetPrefix: args[3],
```

Comment:
what about making the fourth argument optional? 🤔 
as in
```ts
changesetPrefix: args[3] ?? 'dependabot-update'
```

Reply:
Since it is internal there is no real benefit to making it optional. Better to force people to be explicit.